src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,72 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+    private static final StorageDao STORAGE_DAO = new StorageDaoImpl();
+    private static final User VALID_USER = createUser(2123123L, "Ryan Gosling",
+            "JavaIsTheBestProgrammingLanguage", 41);
+    private static final User NULL_ID_USER = createUser(null, "Not Ryan",
+            "PleaseApproveMyPR", 24);
+    private static final User NULL_LOGIN_USER = createUser(12334L, null,
+            "IHateMakingTestsSoMuch", 23);
+    private static final User NULL_PASSWORD_USER = createUser(2141L, "Also Not Ryan",
+            null, 35);
+    private static final User NULL_AGE_USER = createUser(21321L, "%3245##%@",
+            "SoPleaseLetMeFree", null);
+    private static final User ANOTHER_VALID_USER = createUser(2123123L, "Whaaaathup!",
+            "ThisIsATestPassword", 23);
+    private static final User NOT_ADULT_USER = createUser(35252L, "Zorro",
+            "Hazzard!", 14);
+    private static final User SHORT_PASSWORD_USER = createUser(135211L, "Vendetta means",
+            "V", 42);
```

Comment:
We don't need to create users in constants, looks messy.
You can declare users on the class level and initialize them in BeforeAll test